src/utils/ApiClient.js

```diff
@@ -313,9 +318,6 @@ export default class ApiClient {
      * @private
      */
     processEncryptedResponse(httpMethod, err, res, callback) {
-        if (!res) {
```

Comment:
Im assuming this will never occur? of checking 204 status?

Reply:
Yeah it will never occur here now, since this is only called when the content type is application/jose+json and the body is not empty.  If it's a 204 it will go through the regular processResponse and we won't return the error message (which wasn't correct).